index.ts

```diff
@@ -270,6 +271,15 @@ export class Project {
     }
   }
 
+  async binLinks() {
+    await binLinks({ pkg: this.pkg, path: this.baseDir, top: false, global: false });
+  }
+
+  async write() {
```

Comment:
This looks good. We should probably deprecate calling `project.writeSync()` in favor of this. We will need to update the README with new examples of `await project.write()`.

Reply:
in a library like this what the best way to deprecate? console.warn? just documentation with jsdoc @deprecate?